cmd/algons/dnsCmd.go

```diff
@@ -293,26 +318,48 @@ func doDeleteDNS(network string, noPrompt bool, excludePattern string) bool {
 	cloudflareDNS := cloudflare.NewDNS(cfZoneID, cfEmail, cfKey)
 
 	idsToDelete := make(map[string]string) // Maps record ID to Name
+	services := []string{"_algobootstrap", "_metrics"}
+	servicesRegexp, err := regexp.Compile("(algobootstrap|_metrics)._tcp.*algodev.network")
+
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create services expression: %v", err)
+	}
+
+	if includeRegex != nil {
+		services = []string{"N/A when using an include pattern"}
+	}
+
+	for _, service := range services {
+		name := service + "._tcp." + network + ".algodev.network"
+		if includeRegex != nil && network == "" {
+			name = ""
+		}
+
+		records, err := cloudflareDNS.ListDNSRecord(context.Background(), "SRV", name, "", "", "", "")
 
-	for _, service := range []string{"_algobootstrap", "_metrics"} {
-		records, err := cloudflareDNS.ListDNSRecord(context.Background(), "SRV", service+"._tcp."+network+".algodev.network", "", "", "", "")
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Error listing SRV '%s' entries: %v\n", service, err)
+			fmt.Fprintf(os.Stderr, "Error listing SRV entries: %v\n", err)
```

Comment:
I think that we should keep the service, especially for the include use case.

Reply:
It actually doesn't make any sense for the include case. When using cloudflare for `name = ""` it includes both services in one search, so I wanted to avoid calling it twice. I did that by changing the loop from `range {_algobootstrap, _metrics}` to `range { one-unused-service }` in the include case.

I'll try to find a cleaner way to implement that.